com.unity.toonshader/Documentation~/Rimlight.md

```diff
@@ -23,92 +20,87 @@ The Rim Light  highlight the edges of meshes. Rim Light is based on the angle be
 ## Rim Light
 A Check box to enable **Rim Light** that hits the 3D model from behind and emphasizes the contours of the model from the front.
 
-|  Rim Light Off | Rim Light On | 
-| ---- | ---- |
-| <img src="images/RimLightOff.png" height="256"> | <img src="images/RimLightOn.png" height="256"> |
- 
-<br>
+<canvas class="image-comparison" role="img" aria-label="A toon-shaded sphere in a room textured with graphs. The sphere has green and purple bands of color, and specular highlights. With Rim Light enabled, the sphere is bright at the edge.">
+    <img src="images/RimLightOff.png" title="Rim Light disabled">
+    <img src="images/RimLightOn.png" title="Rim Light enabled">
+</canvas>
+<br />
+Drag the slider to compare the images.
 
 ### Rim Light Color
 Specifies the color of **Rim Light**.
 
-<img src="images/RimLightColor.gif" height="256">
-<br>
-
 ### Rim Light Level
 Specifies **Rim Light** intensity.
 
-<img src="images/RimLightLevel.gif" height="256">
+<video title="The same sphere. The intensity of the bright edge grows and shrinks, affecting almost the whole sphere." src="images/RimLightLevel.mp4" width="auto" height="auto" autoplay="true" loop="true" controls></video>
 <br>
 
 ### Adjust Rim Light Area
 Increasing this value narrows the area of influence of **Rim Light**.
 
-<img src="images/AdjustRimLightArea.gif" height="256">
+<video title="The same sphere. The bright light appears at the edge, then the sphere becomes lighter as a whole." src="images/AdjustRimLightArea.mp4" width="auto" height="auto" autoplay="true" loop="true" controls></video>
```

Comment:
I think these two videos won’t look good if they’re not the same size. In the currently published documentation, they’re shown at the same size.